services/core/config.js

```diff
@@ -201,5 +203,5 @@ config.log.gelf = process.env.SERVICE_LOG_GELF || 'false';
 config.log.file = process.env.SERVICE_LOG_FILE || 'false';
 config.log.docker_host = process.env.DOCKER_HOST || 'local';
 config.debug = process.env.SERVICE_LOG_LEVEL === 'debug';
-
+config.defaultDB = 'redisdb';
```

Comment:
What do you think about calling it just `redis`?
I think it corresponds the common usage and the official name is also `Redis`.

Reply:
I would totally agree with you. I will update it to `Redis` now.

Reply:
done!